textalyze.js

```diff
@@ -43,33 +43,46 @@ function stringToCharacters(text) {
  */
 function sanitize(text) {
   if (typeof text === 'string' || text instanceof String) {
-    return text.toLowerCase();
+    return text.toLowerCase().replace(/[^a-zA-Z]+/g, '');
   }
-  return text;
+  return null;
+}
+
+/**
+ * Prints text frequencies histogram
+ * @param {Map} frequencies
+ */
+function printHistogram(frequencies) {
+  const maxWidth = 300;
+  Array.from(frequencies.keys()).sort().forEach((letter) => {
+    const freq = frequencies.get(letter);
+    const freqStr = (freq * 100).toFixed(2).padStart(5, ' ');
+    const bar = '='.repeat(freq * maxWidth);
+    console.log(`${letter} [${freqStr}%] ${bar}`);
+  });
 }
 
 if (require.main === module) {
-  const file = process.argv.length >= 3 ? process.argv[2] : 'sample_data/tale-of-two-cities.txt';
+  const path = process.argv.length >= 3 ? process.argv[2] : 'sample_data/tale-of-two-cities.txt';
```

Comment:
For a user-facing program, I wouldn't use this as a default.  Rather, I'd print a *usage* message explaining to the user how to run the program with a file and then exit with `process.exit(1)`.